tests/test_url_config.py

```diff
@@ -242,3 +242,28 @@ def func2(**kwargs):
         filtered2 = straxen.filter_kwargs(func2, all_kwargs)
         self.assertEqual(filtered2, all_kwargs)
         func2(**filtered2)
+
+    @unittest.skipIf(not straxen.utilix_is_configured(), "No db access, cannot test CMT.")
+    def test_dry_evaluation(self):
```

Comment:
While not being very general, I think the concrete examples are probably more valuable than writing an extensive unit-test